pkg/oauthserver/server/headers/oauthbasic.go

```diff
@@ -0,0 +1,29 @@
+package headers
```

Comment:
Undo this commit - the handler chain changes are too aggressive of a change to backport.